SampleApp/RoutingViewController.swift

```diff
@@ -81,7 +81,7 @@ class RoutingViewController: UIViewController, RoutingSearchDelegate {
 
```

Comment:
Correct. Line 77 in `RoutingViewController` force unwraps the optional which would cause that. The sample app is full of less than stellar error handling :)